src/webui/routes/select.jl

```diff
@@ -0,0 +1,87 @@
+# Step 4: Select a package.
+function select(::HTTP.Request)
```

Comment:
It would be really nice if this application did not require JavaScript. This could be reworked to happen in the backend.

Reply:
Why do we not want JavaScript? We use it here to ping the server for status of the registration. I don't understand how we can do this purely from the backend. The client will timeout if the clone or any other operation takes too long.

Reply:
Plenty of people disable JS when possible, myself included. It's nice to avoid when possible. But we'll leave it as is and handle concurrency differently later.